README.md

```diff
@@ -137,29 +158,31 @@ The below instructions will guide you through each of the steps.
 
 ### DNS
 
-You need to create DNS entries to set up ESS Community. All of these DNS entries must point to your server's IP.
+You need to create DNS A records to set up ESS Community. All of these DNS entries must point to your server's IP.
```

Comment:
I don't think it needs to be `A` records. `CNAMES`  or other provider specific (`ALIAS`/`ANAME`) should work

Reply:
Thank you, wasn't aware. Added A and CNAME as examples.